src/main/resources/config/liquibase/changelog/20211901120000_changelog.xml

```diff
@@ -0,0 +1,117 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-4.1.xsd">
+
+    <!-- More information about the changes in this changelog can be found in the associated PR https://github.com/ls1intum/Artemis/pull/2651 -->
+
+    <!-- Adding a new enum to the exercise table that signifies if and how this exercise counts towards to the course / exam score -->
+    <changeSet author="swaldhauser" id="20211901120000-1">
+        <preConditions onFail="MARK_RAN">
+            <not>
+                <columnExists tableName="exercise" columnName="included_in_overall_score"/>
+            </not>
+        </preConditions>
+
+        <addColumn tableName="exercise">
+            <column name="included_in_overall_score" type="VARCHAR(255)"/>
+        </addColumn>
+    </changeSet>
+
+    <!-- Enum should never be null and all exercises without this enum should be assigned INCLUDED_COMPLETELY -->
+    <changeSet id="20211901120000-2" author="swaldhauser">
+        <addNotNullConstraint tableName="exercise" columnName="included_in_overall_score" columnDataType="VARCHAR(255)" defaultNullValue="INCLUDED_COMPLETELY"/>
+    </changeSet>
+
+    <!-- Default value of the Enum is INCLUDED_COMPLETELY -->
+    <changeSet author="swaldhauser" id="20211901120000-3">
+        <addDefaultValue columnName="included_in_overall_score"
+                         defaultValue="INCLUDED_COMPLETELY"
+                         tableName="exercise" columnDataType="varchar(255)"/>
+    </changeSet>
+
+    <!-- The max points of an exercise should never be null thus null values will be set to 0 -->
+    <changeSet author="swaldhauser" id="20211901120000-4">
+        <sql>
+            UPDATE exercise SET
+            max_score = 0
+            WHERE max_score IS NULL
+        </sql>
+    </changeSet>
+
+    <!-- Adding not null constraint for the max points of an exercise -->
+    <changeSet id="20211901120000-5" author="swaldhauser">
+        <addNotNullConstraint tableName="exercise" columnName="max_score" columnDataType="double" defaultNullValue="0"/>
```

Comment:
I still think that `defaultNullValue="0"` is wrong here and we should rather use `defaultNullValue="1"`
Remember that we do not support 0 point exercises after this PR

Reply:
The defaultNullValue=0 just means that the current exercises that have score=null will be set to score = 0. Setting these to score = 1 makes no sense to me, because we also would not be able to identify them any more as bonus exercises / not included exercises with the subsequent update scripts

Reply:
defaultNullValue does not affect the notNullConstraint it just means what the current Null values in the db should be converted too.

Reply:
ok then please add a comment into the xml file about it :)

Reply:
Done in 385350a23aa047f0deafb3a6bc0920fafaa25681